### PR TITLE
Staging new docs for the new release

### PR DESCRIPTION
**Pull request by baronfel**
This covers inlay hints, launch configurations, tasks, and calls out the code fixes we provide at this time (pending more in-depth docs on these topics, of course)

### CODE REVIEW

**Comment by baronfel** [comment]
Marked this as draft so that we don't publish before the Ionide release occurs.

File: content/Editors/Code/options.md [hunk 1]
```diff
@@ -636,6 +636,30 @@ The prefix displayed before the signature of a pipeline hint
...
+
+Controls if the inlay hints feature is enabled.
+
+**Type:** `boolean`
+
+**Default:** `true`
+
+#### `Fsharp.inlayHints.parameterNames`
+
+Controls if parmaeter-name inlay hints will be displayed for functions and methods.
```
Comment (MangelMaxime) [inline content/Editors/Code/options.md:649]: Typo: `parmaeter` -> `parameter`

File: content/Editors/Code/options.md [hunk 2]
```diff
@@ -636,6 +636,30 @@ The prefix displayed before the signature of a pipeline hint
...
+
+**Type:** `boolean`
+
+**Default:** `true`
+
+#### `FSharp.inlayHints.typeAnnotations`
+
+Controls if type-annotation inlay hitns will be displayed for bindings.
```
Comment (MangelMaxime) [inline content/Editors/Code/options.md:657]: Typo: `hitns` -> `hints`

**Review by MangelMaxime — COMMENTED** [review]
Hello, I found 2 typos while reading the PR

**Comment by baronfel** [comment]
Thanks @MangelMaxime  :)

**Pull request merged**